components/Header/index.tsx

```diff
@@ -32,6 +32,21 @@ const Header = () => {
   const mobileInputRef = useRef<HTMLInputElement>(null);
   const desktopInputRef = useRef<HTMLInputElement>(null);
 
+  const setShowSearch = (showSearch: boolean) => {
+    setSearch((search) => ({
+      ...search,
+      showSearch,
+    }));
+  };
```

Comment:
@ntorbinskiy why this function was even created?
You wrap one setter function, inside another setter function

Reply:
@ntorbinskiy It kind of makes sense to create openSearch, and closeSearch functions, but not this wrapper